spring-cloud-contract-stub-runner/src/test/groovy/org/springframework/cloud/contract/stubrunner/AetherStubDownloaderSpec.groovy

```diff
@@ -58,7 +58,7 @@ class AetherStubDownloaderSpec extends Specification {
 		and:
 			String localRepo = AetherFactories.localRepositoryDirectory(true)
 			new File(localRepo, "org/springframework/cloud/spring-cloud-contract-spec"
-					.replaceAll("/", File.separator)).list().size() > 0
```

Comment:
replaceAll was doing a regex pattern match and not a literal so messed up the test